crates/bevy_pbr/src/light_probe/environment_map.rs

```diff
@@ -0,0 +1,391 @@
+//! Environment maps and reflection probes.
```

Comment:
The old docs contained some info about the split sum approximation, and talked about the use case for view-level envmaps (better quality dynamic indirect lighting compared to AmbientLight). I'd like to keep that info.

Reply:
OK, I merged the text in. Let me know if it looks good.

Reply:
Lgtm, thanks